As we build every rpm as a normal user, not as root, we cant use 
chown here.

It's also no point as the filelist defattr and attr sets them correctly for us.

Signed-off-by: Thomas Backlund <tmb@mandriva.org>

diff -p -up ccstools/Makefile.orig ccstools/Makefile
--- ccstools/Makefile.orig	2009-03-31 21:00:00.000000000 -0300
+++ ccstools/Makefile	2009-04-08 10:08:38.000000000 -0300
@@ -22,7 +22,7 @@ install: all
 	cp -af --remove-destination realpath make_alias domainmatch init_policy.sh tomoyo_init_policy.sh $(INSTALLDIR)/usr/lib/ccs/
 	cp -af --remove-destination $(MISC_FILES) $(INSTALLDIR)/usr/lib/ccs/misc/
 	cp -af --remove-destination README.ccs COPYING.ccs $(INSTALLDIR)/usr/lib/ccs/
-	chown -R root:root ccs-init tomoyo-init $(INSTALLDIR)/usr/lib/ccs/
+	#chown -R root:root ccs-init tomoyo-init $(INSTALLDIR)/usr/lib/ccs/
 	chmod 4711 $(INSTALLDIR)/usr/lib/ccs/misc/force-logout
 	chmod 700 ccs-init tomoyo-init
 	cp -af --remove-destination ccs-init tomoyo-init $(INSTALLDIR)/sbin/
